shunting-yard-emitter/shunting-yard-emitter-kinesis/src/test/java/com/hotels/shunting/yard/emitter/kinesis/listener/KinesisMetaStoreEventListenerTest.java

```diff
@@ -97,7 +98,8 @@ public void onCreateTable() throws Exception {
     when(serializableEvent.getTableName()).thenReturn(TABLE);
     when(serializableListenerEventFactory.create(event)).thenReturn(serializableEvent);
     listener.onCreateTable(event);
-    verify(executorService).submit(messageTask);
+    verify(executorService).submit(captor.capture());
```

Comment:
Since this is covered by tests for Abstract class you can probably just verify that submit() was called with *something* and then wrapper class can be made package protected.

Reply:
done